### DISCUSSION TITLE
tengo una duda sobre adquirir la licencia

### DISCUSSION THREAD

**Discussion by JJGF97** [aclap-dev/video-downloadhelper#489] [Help and Q&A]
### Your question

si compro la licencia, en donde se queda "guardada"? en mi cuenta de firefox? o en donde? es que quiero comprar la licencia pero no se que pase si cambio de computadora

### Addon & CoApp version and details

si compro la licencia, en donde se queda "guardada"? en mi cuenta de firefox? o en donde? es que quiero comprar la licencia pero no se que pase si cambio de computadora

**Comment by GrampaWildWilly** [comment]
You are actually worrying about something that you don't need to worry about.  When you buy a license, you receive an E-mail containing the license key.  You must keep that in a safe place so you can find it on demand at any time.  That E-mail is an important document & you should treat it as you would treat your credit card, your check book, your driver's license, other documents like that.

The license key is just a string of letters & digits.  To find out how to register the license, you need to click the Wiki link at the top of this page.  You should read everything you can find on that page, & you should click all the links on that page & read all of that documentation as well.  For now, there is an explanation on the main Wiki page of how to register a license.  Problem is it's out of date.  Here's something more current:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/defbbe49-34e5-4ebb-a5b1-ff5f8f31c5ec)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/15981473-f8bb-4219-bf49-d332145fe6b0)

That will open another menu that looks something like this:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0afc8d63-3ccb-4af9-9bea-fc9db6542f08)

Now, what you see will not be exactly like this.  I have already registered my license.  (I don't work for VDH.  I'm just another user.)  But what you see should be pretty easy to understand.  Copy/paste your VDH license out of the E-mail you got into the relevant field on this menu.  Then I think you will have a button that says "Register license" or something to that effect.  Click that & you will be all set.

You'll also notice in that image that I have blurred out my license key.  You should NEVER share your license key in public.  Like I said, it's an important document.  You don't go posting your credit card number all over the web, do you?  Same thing for the VDH license.

The license does not reside on any web site . . .  except the VDH web site, where they administer the license data base.  It resides in the VDH settings.  I'm not entirely sure about the internals of how a browser add-on manages its settings & internal storage (VDH being a browser add-on).  But you shouldn't be concerned with that.  It's somewhere on your computer.  I'm an experienced user & I don't know exactly where the license is kept nor how it all works.  I just use VDH & it works.  That's really all any user needs to know.

The VDH license is yours for life.  Not your computer's life.  Your life.  You can use the same license on every computer you own today.  You will be able to use the same license on every computer you may ever own in the future.  But don't share the license with your friends & family.  Make them buy their own licenses.

The reason the Wiki is out of date is that within the past week or so, a new version just came out.  Refer to this discussion: #466.  You should be using that version.

I am encouraged that you found this forum.  You will discover people here who are eager to help.  This is an international cooperative self-help forum populated mainly by your fellow users.  You should read the Wiki, like I already mentioned.  You should get in the habit of referring to it regularly.  It's kind of a user's manual.  You should always try to find an existing discussion that covers any question you might have.  You will save yourself time & effort if you take advantage of what your fellow users have already posted here.  I'm assuming you're a new user.  That means you probably have a lot of reading to do here for the time being.  But at a certain point, you'll have learned enough that your reading time will decrease.

VDH employees also come in here all the time.  @jcfrog Jérôme handles questions about licenses & answers many usage questions.  @paulrouget Paul is doing the active development on VDH.  He will deal with any bugs that come up.  He's been with VDH for barely a year & I must say, he knows everything there is to know about VDH.  The sheer volume of change in VDH we've had here from Paul is probably more than we've had in the preceding 5 years combined.

You've picked a good time to join the community.  Welcome!

**Reply by JJGF97** [reply]
damn bro, I mean, thanks a lot, you answer all the questions i had, thanks again.
